pkg/cimfs/cim_writer_windows.go

```diff
@@ -210,15 +291,14 @@ func (c *CimFsWriter) Close() error {
 	if err := c.commit(); err != nil {
 		return &OpError{Cim: c.name, Op: "commit", Err: err}
 	}
-	if err := winapi.CimCloseImage(c.handle); err != nil {
-		return &OpError{Cim: c.name, Op: "close", Err: err}
-	}
+	winapi.CimCloseImage(c.handle)
 	c.handle = 0
 	return nil
 }
 
-// DestroyCim finds out the region files, object files of this cim and then delete
-// the region files, object files and the <layer-id>.cim file itself.
+// DestroyCim finds out the region files, object files of this cim and then delete the
+// region files, object files and the <layer-id>.cim file itself.  Note that this only
+// applies to forked CIMs. For block CIMs `os.Remove` should be enough.
 func DestroyCim(ctx context.Context, cimPath string) (retErr error) {
```

Comment:
This seems dangerous as it could delete files that another cim relies on. Can you add a note to the function comment explaining that?